kuma/wiki/models.py

```diff
@@ -568,6 +568,10 @@ def build_json_data(self):
         if self.pk:
             for translation in self.other_translations:
                 revision = translation.current_revision
+                if revision.summary:
+                   summary = revision.summary
+                else:
+                   summary = translation.get_summary(strip_markup=False)
```

Comment:
Below, we do a `if self.current_revision.summary` check before we call `self.get_summary()`. I notice the main change here is to remove it?

Could we uplift the same check here? I.e., `if translation.current_revision: summary = translation.current_revision.summary; else summary = translation.get_summary()`? I'd just like to call `get_summary()` as little as possible.